chapter01/introduction_overview_of_cuqipy.md

```diff
@@ -101,29 +101,13 @@ See setting up a Bayesian model in CUQIpy in 4 steps [here](https://cuqi-dtu.git
 - [Chao Zhang (Charlie)](https://www.dtu.dk/english/person/chao-zhang?id=207508&entity=profile)
 
 
-### Remarks <a class="anchor" id="remarks"></a>
-- This is first time we offer the CUQIpy component of the course.
-- We are excited about this component and hope you will find it useful.
-- I am here to facilitate your learning of this component.
-- Feel free to ask questions anytime.
-- Feel free to provide feedback anytime and help us improve.
-- There will be a lot of hands-on exercises and examples to help you understand the concepts.
-- You are encouraged to work in pairs or groups of 3 to discuss and solve the exercises.
-- Disclaimer: keep in mind that notation may vary among different course components.
-- Throughout we will introduce many tools in CUQIpy and discuss 
-    - how to use them
-    - for what purpose
-    - and identifying the right tool
-- Throughout all the course components, including CUQIpy, you will end up with more tools in your BIP toolbox!
-![toolbox](../images/toolbox3.jpeg)
-
 
 ### How to run the code and solve the coding exercises in this mini-book? <a class="anchor" id="running-the-code"></a>
```

Comment:
Just a thought: this is a vital piece of information for using this book, and it's a pit that it is somehow hidden in a section called "Overview of CUQIpy". Do you think it deserves a more noticeable place, for example somewhere before chapter01?

Reply:
Thank you @chaozg, this is a good point, I made a note of it in #68